appveyor/install_kiwix-tools.cmd

```diff
@@ -1,4 +1,4 @@
 REM ========================================================
 REM Install kiwix-tools
 curl -fsSL -O https://download.kiwix.org/release/kiwix-tools/kiwix-tools_win-i686-3.1.0-1.zip || exit /b 1
-7z e kiwix-tools_win-i686-3.1.0.zip -o%EXTRA_DIR%\bin kiwix-serve.exe || exit /b 1
+7z e kiwix-tools_win-i686-3.1.0-1.zip -o%EXTRA_DIR%\bin kiwix-serve.exe || exit /b 1
```

Comment:
This is a line I forget to change in the previous commit.
I have just push a new commit on master to fix this.